### DISCUSSION TITLE
Check if account is validated

### DISCUSSION THREAD

**Discussion by develocode777** [serverpod/serverpod#2016] [Q&A]
On login I would like to check if the user has validated his account with the code sent to his email and if not show a message to check his email for the code, but I don't find find a way to check if he has validated his account.

**Comment by Metaildust** [comment]
i think there's not a built-in method to login in using a v-code. So there's only using password to login if the account is registered using email. However, you can make a method  to achive it your own using endpoint. i think they will create this method in the future :)

**Reply by develocode777** [reply]
I am using email and password but I want to cover the case when the user has not validated the email.

**Reply by Metaildust** [reply]
but as what i've know if you want to register, you'll have to valid the email, here's the code how register work:

final authController = EmailAuthController(client.modules.auth);
await authController.createAccountRequest(
                                    userName.text,
                                    emailAddress.text,
                                    password.text);
await authController .validateAccount(emailAddress.text,
                                                             vCode.text);
                                                             
follow this: https://medium.com/serverpod/getting-started-with-serverpod-authentication-part-1-72c25280e6e9

**Reply by Metaildust** [reply]
and if you want the user to valid after login to make it more safe, you'll have to write your own because there's not a built-in method
